src/adapter/src/catalog/storage/stash.rs

```diff
@@ -228,6 +228,30 @@ pub async fn initialize(
                 privileges: AclMode::USAGE,
             },
         ),
+        (
+            DefaultPrivilegesKey {
+                role_id: RoleId::Public,
```

Comment:
`role_id` is the role doing the granting, not the role gain the privilege, right?

Reply:
That's correct. That's the name PostgreSQL uses in their docs, though I'd be open to changing it to something more obvious.